src/progressbar/progressbar.js

```diff
@@ -32,6 +38,15 @@ angular.module('ui.bootstrap.progressbar', [])
     this.removeBar = function(bar) {
         this.bars.splice(this.bars.indexOf(bar), 1);
     };
+
+    $scope.$watch('max', (function(self){
```

Comment:
This block is a bit awkward - why not use the `self` variable defined at the top of the controller?